[2022-05-31 19:15:49,261] {taskinstance.py:1160} INFO - Dependencies all met for <TaskInstance: fhvhv_ingestion_gcs_dag.download_dataset_task scheduled__2020-07-01T06:00:00+00:00 [queued]>
[2022-05-31 19:15:49,342] {taskinstance.py:1160} INFO - Dependencies all met for <TaskInstance: fhvhv_ingestion_gcs_dag.download_dataset_task scheduled__2020-07-01T06:00:00+00:00 [queued]>
[2022-05-31 19:15:49,345] {taskinstance.py:1357} INFO - 
--------------------------------------------------------------------------------
[2022-05-31 19:15:49,347] {taskinstance.py:1358} INFO - Starting attempt 1 of 1
[2022-05-31 19:15:49,351] {taskinstance.py:1359} INFO - 
--------------------------------------------------------------------------------
[2022-05-31 19:15:49,461] {taskinstance.py:1378} INFO - Executing <Task(BashOperator): download_dataset_task> on 2020-07-01 06:00:00+00:00
[2022-05-31 19:15:49,529] {standard_task_runner.py:52} INFO - Started process 729 to run task
[2022-05-31 19:15:49,564] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'fhvhv_ingestion_gcs_dag', 'download_dataset_task', 'scheduled__2020-07-01T06:00:00+00:00', '--job-id', '471', '--raw', '--subdir', 'DAGS_FOLDER/fhv_ingestion.py', '--cfg-path', '/tmp/tmp971lz59v', '--error-file', '/tmp/tmptz33fhr1']
[2022-05-31 19:15:49,571] {standard_task_runner.py:80} INFO - Job 471: Subtask download_dataset_task
[2022-05-31 19:15:50,053] {task_command.py:370} INFO - Running <TaskInstance: fhvhv_ingestion_gcs_dag.download_dataset_task scheduled__2020-07-01T06:00:00+00:00 [running]> on host 3f188d455d5a
[2022-05-31 19:15:50,254] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:202: AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
  warnings.warn(_create_deprecation_warning(key, self._deprecation_replacements[key]))

[2022-05-31 19:15:50,476] {taskinstance.py:1572} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=fhvhv_ingestion_gcs_dag
AIRFLOW_CTX_TASK_ID=download_dataset_task
AIRFLOW_CTX_EXECUTION_DATE=2020-07-01T06:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2020-07-01T06:00:00+00:00
[2022-05-31 19:15:50,483] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-05-31 19:15:50,495] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'curl -sSL https://s3.amazonaws.com/nyc-tlc/trip+data/fhvhv_tripdata_2020-07.parquet > /opt/***/output_fhvhv_2020-07.parquet']
[2022-05-31 19:15:50,554] {subprocess.py:85} INFO - Output:
[2022-05-31 19:16:10,119] {local_task_job.py:221} WARNING - State of this instance has been externally set to failed. Terminating instance.
[2022-05-31 19:16:10,131] {process_utils.py:129} INFO - Sending Signals.SIGTERM to group 729. PIDs of all processes in the group: [730, 731, 729]
[2022-05-31 19:16:10,133] {process_utils.py:80} INFO - Sending the signal Signals.SIGTERM to group 729
[2022-05-31 19:16:10,142] {taskinstance.py:1542} ERROR - Received SIGTERM. Terminating subprocesses.
[2022-05-31 19:16:10,145] {subprocess.py:103} INFO - Sending SIGTERM signal to process group
[2022-05-31 19:16:10,235] {process_utils.py:75} INFO - Process psutil.Process(pid=729, status='terminated', exitcode=0, started='19:15:49') (729) terminated with exit code 0
[2022-05-31 19:16:10,237] {process_utils.py:75} INFO - Process psutil.Process(pid=730, status='terminated', started='19:15:50') (730) terminated with exit code None
[2022-05-31 19:16:10,240] {process_utils.py:75} INFO - Process psutil.Process(pid=731, status='terminated', started='19:15:50') (731) terminated with exit code None
